ibm_catalog.json

```diff
@@ -0,0 +1,501 @@
+			"long_description": "Simplify your IBM Cloud Account onboarding to Apptio Cloudability with this deployable architecture. Only an IBM Cloud IAM Api Key and Cloudability API key are required. This deployable architecture will take care of all of the necessary steps to get your IBM Cloud billing data into Apptio Cloudability. This includes the following:\n\n1. Creating Key Protect Encrypted COS Buckets to store your daily billing snapshots.\n2. Setup writing of billing snapshots to the COS Bucket.\n3. Grant Apptio Cloudability access to read from the COS buckets\n4. Add your IBM Cloud account to Cloudability and run access verification from Cloudability.\n\nYour IBM Cloud billing data will begin showing up in Apptio cloudability the next day. \nTo obtain your Apptio Cloudability API key visit https://app.apptio.com/cloudability#/settings/preferences under the section **Cloudability API** select **Enable API** which will generate an api key. If you don't have a Cloudability account start your tree trial now: https://www.apptio.com/products/cloudability/#cldy-trial.",
+			"provider_name": "apptio-cloudability",
+			"offering_docs_url": "https://cloud.ibm.com/docs/billing-usage?topic=billing-usage-exporting-your-usage",
+			"offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-apptio-cloudability-onboarding/main/images/Apptio_logo.svg",
+			"features": [
+				{
+					"title": "Daily exports of your cloud billing reports to a COS Bucket",
+					"description": "Cloudability reads CSV Billing reports which are added to a COS bucket daily for each account in your enterprise."
+				},
+				{
+					"title": "Encrypted Bucket with Key Protect",
+					"description": "Daily billing reports are stored in a Key Protect encrypted bucket allowing the automated rotatation of encryption keys. See https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-tutorial-kp-encrypt-bucket."
+				},
+				{
+					"title": "Least Privileged Operations to Cloudability",
+					"description": "Custom access roles are created so that cloudability only has the minimal required access to your IBM Cloud Account."
+				},
+				{
+					"title": "Cloudability Onboarding Verification",
+					"description": "Verification of the account onboarding to Apptio Cloudability is performed so that you have confidence that the configuration is correct."
+				},
+				{
+					"title": "Add all accounts in an Enterprise to Cloudability",
+					"description": "Running this deployable architecture at the root (primary) Enterprise Account will cause all existing and future accounts in the enterprise to be added to Apptio Cloudability."
+				},
+				{
+					"title": "Managed with Code!",
+					"description": "Easily re-configure or clean-up the onboarding of your accounts to Cloudability and ensure that everything is centrally connected and configured."
+				}
+			],
+			"flavors": [
+				{
+					"label": "all-inclusive",
```

Comment:
The label is the display name in the UI - perhaps make it more human readable value (you can have spaces in the value)